t/test_paths.pm

```diff
@@ -34,12 +39,13 @@ BEGIN {
     $pf::file_paths::firewall_sso_config_file = catfile($test_dir,'data/firewall_sso.conf');
     $pf::file_paths::config_file = catfile($test_dir,'data/pf.conf');
     $pf::file_paths::pf_config_file = catfile($test_dir,'data/pf.conf');
```

Comment:
It would be better to only override pf.conf if an environmental variable is set.
So it can be use our own special pf.conf for testing.